samza-test/src/test/java/org/apache/samza/test/processor/TestStreamProcessor.java

```diff
@@ -132,7 +132,7 @@ public void testStreamProcessorWithNoTask() {
     configMap.remove("task.class");
     final Config configs = new MapConfig(configMap);
 
-    StreamProcessor processor = new StreamProcessor(1, configs, new HashMap<>());
```

Comment:
I couldn't add unit test because the callback in invoke inside a Runnable in SamzaContainerController. And, that class uses a static method to create the container instance. So, at this point, I don't know how to add the unit test without extensive surgery to the existing code :) if you have any suggestions, please let me know :)